e2e/pages/createClaim/uploadParticularsOfClaim.page.js

```diff
@@ -6,19 +6,13 @@ module.exports = {
 
   fields: {
     servedDocumentFiles: {
-      particularsOfClaim: '#servedDocumentFiles_particularsOfClaimDocument',
       options: [
-        '#servedDocumentFiles_scheduleOfLoss',
-        '#servedDocumentFiles_certificateOfSuitability',
+        '#servedDocumentFiles_particularsOfClaimFile',
```

Comment:
you can add back the other options, I think the code cycles over them, better to test more fields